guard/src/rules/eval_context_tests.rs

```diff
@@ -28,11 +28,7 @@ fn extraction_test() -> Result<()> {
             .rules
             .get("aws_route53_recordset")
             .map(|s| s.get(0))
-            .map(|s| match s {
-                Some(r) => Some(*r),
-                None => None,
-            })
-            .flatten(),
+            .and_then(|s| s.copied()),
```

Comment:
Wait, is there a logic change here?

Reply:
nope, see: https://doc.rust-lang.org/rust-by-example/error/option_unwrap/and_then.html